e2e/components/Axe.test.ts

```diff
@@ -0,0 +1,49 @@
+import {test, expect} from '@playwright/test'
+// @ts-ignore since this file is generated in the ci to generate this file run npx build storybook in packages/react
+import componentsConfig from '../../packages/react/storybook-static/index.json'
+import {visit} from '../test-helpers/storybook'
+import {themes} from '../test-helpers/themes'
+
+/**
+ * These stories should not be tested in the CI because they are stress-tests and
+ * perform slowly
+ */
+const SKIPPED_TESTS = [
+  'components-treeview-features--stress-test',
+  'components-treeview-features--contain-intrinsic-size',
+]
+
+type Component = {
+  name: string
+}
+
+const {entries} = componentsConfig
+
+test.describe('@aat', () => {
```

Comment:
```suggestion
test.describe('aat', () => {
```

I would just annotate this as `aat`, or remove it, since we're using the `@aat` tag at the test-level to filter things out currently (although we could totally change this in the future.

Reply:
My thought was to remove all `@aat` tests after this is merged since we don't need to generate them ourselves

Reply:
@joshblack do you feel like we need the other @aat tests now?

Reply:
@kendallgassner I totally bet we could, especially the automated/boilerplate ones.

I think it can still be helpful to have this tag since there are stories where we would like to put the component in a specific state and then run axe, ideally. Like with menus to make sure we test things when it's open and closed or if we want to use the `matrix` test helper to make sure we run axe against prop combinations of things.

Reply:
If we keep the '@aat' in this file, doesn't it break up the tests to run concurrently in [aat-report](https://github.com/primer/react/actions/runs/13019158341/job/36315594252?pr=5592). I assumed aat-runner 1-4 were different threads.

Reply:
@kendallgassner I think we should definitely keep `@aat` in the file for the reasons you mentioned, just was saying that we could add it at the test-level instead of in the `describe` (which seems like the case on line 38? let me know if I'm misreading 👀)

If we have that in place then I think we can remove the outer `describe` block or replace it with just `describe('aat')`